sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -66,19 +67,45 @@ public async Task RunSamplesAsync()
 
             basicDigitalTwin.CustomProperties.Add("Component1", componentMetadata);
 
-            string dt1Payload = JsonSerializer.Serialize(basicDigitalTwin);
+            string basicDtPayload = JsonSerializer.Serialize(basicDigitalTwin);
 
-            Response<string> createDt1Response = await DigitalTwinsClient.CreateDigitalTwinAsync(dtId1, dt1Payload).ConfigureAwait(false);
-            Console.WriteLine($"Created digital twin {dtId1} with response {createDt1Response.GetRawResponse().Status}.");
+            Response<string> createBasicDtResponse = await DigitalTwinsClient.CreateDigitalTwinAsync(basicDtId, basicDtPayload).ConfigureAwait(false);
+            Console.WriteLine($"Created digital twin {basicDtId} with response {createBasicDtResponse.GetRawResponse().Status}.");
 
             #endregion Snippet:DigitalTwinsSampleCreateBasicTwin
 
+            // You can also get a digital twin and deserialize it into a BasicDigitalTwin.
+            // It works well for basic stuff, but as you can see it gets more difficult when delving into
+            // more complex properties, like components.
+
+            #region Snippet:DigitalTwinsSampleGetBasicDigitalTwin
+
+            Response<string> getBasicDtResponse = await DigitalTwinsClient.GetDigitalTwinAsync(basicDtId).ConfigureAwait(false);
+            if (getBasicDtResponse.GetRawResponse().Status == (int)HttpStatusCode.OK)
+            {
```

Comment:
What about else scenarios? Don't we want used to know it failed.

Reply:
If I had something compelling to put in the sample for that, I would. We have lots of other examples of errors in other places. I tried to sculpt this sample best for the doc comment with transform to web documentation in mind.